.github/workflows/back-tests.yml

```
@@ -0,0 +1,62 @@
+        node-version: '16' # Specify the Node.js version you are using
+
+    - name: Install dependencies
+      working-directory: junqo_back
+      run: npm install
+
+    - name: Run tests
+      working-directory: junqo_back
+      run: npm test
+:bulb: Codebase verification
+
+Update Flutter version in GitHub Actions workflow to meet project requirements.
```

actionlint

could not parse as YAML: yaml: line 35: could not find expected ':'

(syntax-check)